packages/synapse-react-client/src/components/DataGrid/columns/AutocompleteColumn.tsx

```diff
@@ -100,8 +120,23 @@ export function AutocompleteCell({
         '& .MuiAutocomplete-inputRoot': {
           padding: '0 10px',
           backgroundColor: 'inherit',
+          pointerEvents: focus ? undefined : 'none',
         },
         '& .MuiFormControl-root': { height: '100%' },
+        '& .MuiAutocomplete-clearIndicator': {
```

Comment:
Styles for the clear and popup icons. Hide by default except when active or on hover.